build-aux/snap/snapcraft.yaml

```diff
@@ -28,172 +29,272 @@ license: GPL-3.0
 #
 # See the comments from jdstrand in
 # https://forum.snapcraft.io/t/5547/10
+
 parts:
-  snapd-deb:
+  dynamic-linker:
     plugin: nil
-    source: .
-    build-snaps: [go/1.18/stable]
-    # these packages are needed to call mkversion.sh in override-pull, all other
-    # dependencies are installed using apt-get build-dep
     build-packages:
-      - git
-      - dpkg-dev
-    after: [apparmor]
-    override-pull: |
-      snapcraftctl pull
-      # set version, this needs dpkg-parsechangelog (from dpkg-dev) and git
-      snapcraftctl set-version "$(./mkversion.sh --output-only)"
-      # Ensure that ./debian/ packaging which we are about to use
-      # matches the current `build-base` release. I.e. ubuntu-16.04
-      # for build-base:core, etc.
-      ./generate-packaging-dir
-      # install build dependencies
-      export DEBIAN_FRONTEND=noninteractive
-      export DEBCONF_NONINTERACTIVE_SEEN=true
-      sudo -E apt-get build-dep -y ./
-      ./get-deps.sh --skip-unused-check
+      - gawk
+      - bison
+    source: https://git.launchpad.net/ubuntu/+source/glibc
+    source-branch: ubuntu/jammy-updates
```

Comment:
Should this be `jammy-security`?

Reply:
Does not `ubuntu/jammy-updates` contain also security fixes?

Reply:
Also it is the same commit.

Reply:
jammy-updates can from time to time contain regressions and/or changes that are not security relevant. Previously all snapd updates were built against security.

It's fine to choose either, it's just likely tracking -updates will generate and require more rebuilds of snapd. However, snapd releases more often than glibc SRUs, so fairly moot.

Reply:
stage-packages come from -updates so I think it makes sense for glibc to also come from jammy-updates to keep things as consistent as possible.